articles/install-guide/csinstall.md

```diff
@@ -135,7 +136,86 @@ Of course, you can also build and run Q# programs from the command line by simpl
 
         You should see the following output: `Hello quantum world!`
 
+## Q# with other .NET languages <a name="dotnet"></a>
+
+Q# is compiled to C# for the purposes of local simulation. This makes them very easy to combine. However, you can also use other .NET languages like F# or VB\.NET, although the process is less straightforward.
+
+### Develop with Q# + F# creating a Q# library
+
+You can't mix Q# and F# files like we do with Q# and C#. To combine F# with Q# you need to create a Q# library which will yield a .csproj project with only Q# files in it. Then you need to reference it from a purely F# application.
```

Comment:
This isn't quite as big a limitation as it sounds, in that .NET generally doesn't allow mixing languages inside the same project. C# and Q# are kind of an exception at the moment; even to get C# and F# to play along, you need two separate projects.
```suggestion
You can't mix Q# and F# files in the same project like we do with Q# and C#. To combine F# with Q# you need to create a Q# library which will yield a .csproj project with only Q# files in it. Then you need to reference it from a purely F# application.
```